src/containers/Login/Login.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from "react";
+import Input from "../../components/Input/Input";
+import "./styles/LoginStyle.css";
+import { connect } from "react-redux";
+import { apiCalls } from "../../Helpers/axios";
+import { ShowMessage } from "../../redux/actions/SnackBarAction";
+import logout from "../../Helpers/logout";
+export class UnconnectedLogin extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      user: {
+        email: "",
+        password: ""
+      }
+    };
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+  }
+  componentDidMount() {
+    logout();
```

Comment:
I was thinking that it would be better if a user is already logged into the app to be redirected to the homepage instead of being logged out

Reply:
Well received, Implemented a redirect to the homepage in case a user is logged in.